tests/enterprise-contract/contract.go

```diff
@@ -87,32 +75,17 @@ var _ = framework.EnterpriseContractSuiteDescribe("Conforma E2E tests", Label("e
 		BeforeAll(func() {
 			buildPipelineRunName = fmt.Sprintf("buildah-demo-%s", util.GenerateRandomString(10))
 			image = fmt.Sprintf("quay.io/%s/test-images:%s", utils.GetQuayIOOrganization(), buildPipelineRunName)
-			sharedSecret, err := fwk.AsKubeAdmin.CommonController.GetSecret(constants.QuayRepositorySecretNamespace, constants.QuayRepositorySecretName)
-			Expect(err).ShouldNot(HaveOccurred(), fmt.Sprintf("error when getting shared secret - make sure the secret %s in %s namespace is created", constants.QuayRepositorySecretName, constants.QuayRepositorySecretNamespace))
 
-			_, err = fwk.AsKubeAdmin.CommonController.GetSecret(namespace, constants.QuayRepositorySecretName)
-			if err == nil {
-				err = fwk.AsKubeAdmin.CommonController.DeleteSecret(namespace, constants.QuayRepositorySecretName)
-				Expect(err).ToNot(HaveOccurred())
-			} else if !k8sErrors.IsNotFound(err) {
-				Expect(err).ToNot(HaveOccurred())
-			}
-
-			repositorySecret := &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: constants.QuayRepositorySecretName, Namespace: namespace},
-				Type: corev1.SecretTypeDockerConfigJson,
-				Data: map[string][]byte{corev1.DockerConfigJsonKey: sharedSecret.Data[".dockerconfigjson"]}}
-			_, err = fwk.AsKubeAdmin.CommonController.CreateSecret(namespace, repositorySecret)
-			Expect(err).ShouldNot(HaveOccurred())
-			err = fwk.AsKubeAdmin.CommonController.LinkSecretToServiceAccount(namespace, constants.QuayRepositorySecretName, constants.DefaultPipelineServiceAccount, true)
```

Comment:
not sure delete LinkSecretToServiceAccount will cause failure or not.

Reply:
it shouldn't, because it uses `fwk.AsKubeAdmin.CommonController.CreateQuayRegistrySecret` method instead :) see below